_posts/2022/2022-10-03-date-and-time-extensions-are-an-unsung-hero-of-active-support.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Time extensions are an unsung hero of Active Support"
+description: "There’s a method for that"
+layout: article
+category: ruby
+image:
+  base: "2022/time-ranges-are-an-unsung-hero-of-active-support"
+  alt: "Calendar with pins"
+  credit: "Towfiqu barbhuiya"
+  source: "https://unsplash.com/photos/bwOAixLG0uc"
+
+---
+
+In Active Support‘s extensions to the core Ruby classes, some of the most useful and plentiful are related to `Time`.
+
+One of Rails’ founding uses was to provide helpful methods to provide reusable methods for regularly performed tasks. In our web applications we‘re often using time periods.
```

Comment:
*Rails's

(apostrophe 's' for singulars!)

Reply:
Repetition of "to provide" here feels a little clunky.

Reply:
```suggestion
One of Rails’s founding uses was to provide helpful, reusable methods for regularly performed tasks. In our web applications we‘re often using time periods.
```